app/views/contacts/_extra_questions.html.erb

```diff
@@ -21,4 +21,15 @@
     <%= form.radio_button :any_children_below_15, false, :id => "any_children_below_15_false" %>
     <label for="any_children_below_15_false" class="radio__label">No</label>
   </div>
+
+  <legend><h2 class="field-section__title">Are you or anyone in your home showing any symptoms of COVID-19?</h2></legend>
```

Comment:
These 3 questions should actually be wrapped in their own <fieldset> tags. that's why the display is a bit funky. Not a huge deal though.